test/tsconfig.json

```diff
@@ -9,7 +9,7 @@
         "dom",
         "es7"
       ],
-      "types": ["node"],
+      "types": ["node", "openfin"],
```

Comment:
@tomrobertsOF I needed to add this to get the tests to build after merging. Not sure how this wasn't required earlier?..

I get these won't make sense for unit tests, but should be required for the integration tests to build?..

Reply:
See you've already fixed, but for everyone else's benefit: nothing in tests should have any dependence (directly or indirectly) on `client/connection.ts`. Also, for this case specifically, `client/internal.ts` is intended to be the bottom of the dependency tree exporting only types and constants and having no dependencies of its own.